StoreMessageContentActivity/handler.ts

```diff
@@ -171,51 +167,49 @@ const getServicePreferenceValueOrError = (
   ServicePreferenceError,
   ReadonlyArray<BlockedInboxOrChannelEnum>
 > =>
-  taskEither
-    .of<ServicePreferenceError, ServicesPreferencesMode>(
+  pipe(
+    TE.of<ServicePreferenceError, ServicesPreferencesMode>(
```

Comment:
Do we need to write explicit types here?

Reply:
Unfortunately in this cases TS cannot infer the Left type (since it's not defined), so we need to explicit it